nessie/golden/lakectl_doctor_with_long_length_access_key_id.golden

```diff
@@ -0,0 +1,6 @@
+access_key_id value looks suspicious...
+It looks like you have a problem with your '.lakectl.yaml' file. 
+It is possible that the access_key_id' or 'secret_access_key' you supplied are wrong 
+
+
+Error executing command.
```

Comment:
Same as [this comment](https://github.com/treeverse/lakeFS/pull/2948/files#r806420841), only for the access_key_id. Should create a single golden file, e.g. `lakectl_doctor_invalid_access_key_id` and use it for all relevant cases